test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,51 @@
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+
+chai.use(chaiSubset);
+
+const urlBase = 'https://api.github.com';
+const githubUserName = 'aperdomob';
+const company = 'Perficient Latam';
+const name = 'Alejandro Perdomo';
+const repoName = 'jasmine-json-report';
+const desc = 'A Simple Jasmine JSON Report';
+
+describe('Github Api Test', () => {
+  it('GET method consumption', async () => {
+    const response = await axios.get(`${urlBase}/users/${githubUserName}`);
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.data.login).to.equal(githubUserName);
+    expect(response.data.company).to.equal(company);
+    expect(response.data.name).to.equal(name);
+  });
+  it('GET method for repos', async () => {
+    const repos = await axios.get(`${urlBase}/users/${githubUserName}/repos`);
+    const repository = repos.data.find((element) => element.name === repoName);
+    expect(repos.status).to.equal(StatusCodes.OK);
+    /* Expect name of repository jasmine-json-report */
+    expect(repository.name).to.equal(repoName);
+    expect(repository.private).to.equal(false);
+    expect(repository.description).to.equal(desc);
+  });
+
+  it('GET zip file repository', async () => {
+    /* Download zip file repo https://api.github.com/repos/aperdomob/jasmine-json-report/zipball/master */
+    const repos = await axios.get(`${urlBase}/repos/${githubUserName}/${repoName}/zipball/master`);
```

Comment:
the repository response of the previous test (which you store in the repository variable) can be used to get this URL (if you use `repo.url`):
![image](https://user-images.githubusercontent.com/10209180/176711611-92f3d2a4-def3-44be-8bbf-837f36507eb0.png)